ftml/src/parsing/rule/impls/block/blocks/include.rs

```diff
@@ -33,7 +33,7 @@ pub const BLOCK_INCLUDE: BlockRule = BlockRule {
     name: "block-include",
     accepts_names: &["include"],
     accepts_special: false,
-    newline_separator: true,
+    accepts_newlines: true,
```

Comment:
Just noting that I'm curious how this will work for blocks such as include, which to my knowledge doesn't have a concept of a head or tail, which is what I assumed `accepts_newlines` is for - it just has a single tag

Reply:
Do you have any examples other than `[[include]]`? Includes are a special case that are replaced before the preprocessor step.

But if you mean head-only blocks like `[[module Rate]]`, it will try to consume a newline after, then stop because there's no body to fetch.

Reply:
That's a good point regarding include. The rating module is also the only example I had in mind, I guess there's also [[newlines n]].

I think that default response - consuming a newline if possible and stopping there - is fine for cases like these.